android/build.gradle

```diff
@@ -2,14 +2,15 @@ group 'com.whelksoft.flutter_native_timezone'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.6.10'
```

Comment:
I'm just trying if I can approve this as an outsider...

Reply:
> I'm just trying if I can approve this as an outsider...

Unfortunately it can't :(